test_scripts/RC/Capabilities/001_All_modules_all_params.lua

```diff
@@ -0,0 +1,34 @@
+---------------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SDL_RC] Capabilities
+--
+-- Description:
+-- In case:
+-- 1) SDL gets all RC capabilities for RADIO and CLIMATE modules through RC.GetCapabilities
+-- SDL must:
+-- 1) Send RPC request to HMI and resend HMI answer to Mobile app
+---------------------------------------------------------------------------------------------------
+--[[ Required Shared libraries ]]
+local runner = require('user_modules/script_runner')
+local commonRC = require('test_scripts/RC/commonRC')
+--[[ Local Variables ]]
+local modules = { "CLIMATE", "RADIO" }
+
+--[[ Scenario ]]
+runner.Title("Preconditions")
+runner.Step("Clean environment", commonRC.preconditions)
+runner.Step("Start SDL, HMI (HMI has all posible RC capabilities), connect Mobile, start Session", commonRC.start,
+	{commonRC.buildHmiRcCapabilities(commonRC.DEFAULT, commonRC.DEFAULT, commonRC.DEFAULT)})
+runner.Step("RAI, PTU", commonRC.rai_ptu)
+runner.Step("Activate App1", commonRC.activate_app)
+
+runner.Title("Test")
+
+for _, mod in pairs(modules) do
+  runner.Step("GetInteriorVehicleData " .. mod, commonRC.subscribeToModule, { mod, 1 })
+  runner.Step("SetInteriorVehicleData " .. mod, commonRC.rpcAllowed, { mod, 1, "SetInteriorVehicleData" })
+  runner.Step("ButtonPress " .. mod, commonRC.rpcAllowed, { mod, 1, "ButtonPress" })
+end
+
+runner.Title("Postconditions")
+runner.Step("Stop SDL", commonRC.postconditions)
```

Comment:
@Itileda New line at the end of file is present in all scripts. 
GitHub hides it on review.